test/build_test.go

```diff
@@ -217,3 +224,32 @@ RUN uname -a
 	sb.WriteLogFile(b, "buildx", []byte(out))
 	require.NoError(b, err, out)
 }
+
+func benchmarkBuildExportUncompressed(b *testing.B, sb testutil.Sandbox) {
+	benchmarkBuildExport(b, sb, "uncompressed")
+}
+func benchmarkBuildExportGzip(b *testing.B, sb testutil.Sandbox) {
+	benchmarkBuildExport(b, sb, "gzip")
+}
+func benchmarkBuildExportEstargz(b *testing.B, sb testutil.Sandbox) {
+	benchmarkBuildExport(b, sb, "gzip")
+}
+func benchmarkBuildExportZstd(b *testing.B, sb testutil.Sandbox) {
+	benchmarkBuildExport(b, sb, "zstd")
```

Comment:
`zstd` compression in available since v0.10.0 so it fails for older refs like: https://github.com/moby/buildkit-bench/actions/runs/10995084553/job/30525242309?pr=146#step:8:139

```
BenchmarkBuild/BenchmarkBuildExportZstd/ref=v0.9.0/run=1
    build_test.go:254: 
        	Error Trace:	/src/test/build_test.go:254
        	            				/src/test/build_test.go:238
        	            				/src/util/testutil/run.go:127
        	            				/src/util/testutil/run.go:253
        	            				/src/util/testutil/run.go:258
        	            				/src/util/testutil/run.go:280
        	            				/usr/local/go/src/testing/benchmark.go:193
        	            				/usr/local/go/src/testing/benchmark.go:215
        	            				/usr/local/go/src/runtime/asm_amd64.s:1695
        	Error:      	Received unexpected error:
        	            	exit status 1
        	Test:       	BenchmarkBuild/BenchmarkBuildExportZstd/ref=v0.9.0/run=1
        	Messages:   	ERROR: failed to solve: unsupported layer compression type: zstd
```

I will disable this benchmark for now but we could solve this with https://github.com/moby/buildkit-bench/issues/120 or patches if that makes sense